pkg/api/types.go

```diff
@@ -1595,6 +1595,45 @@ type Handler struct {
 	TCPSocket *TCPSocketAction
 }
 
+// DeleteExecAction describes a "run in container" action that will be invoked
+// inside of a container prior to sending the TERM signal to the container's
```

Comment:
Can you please elaborate on what the shared PID namespace implies here? How does it relate to the termination procedure?